src/App.js

```diff
@@ -26,8 +26,8 @@ import PageviewTracker, {
   EventCategory,
 } from 'components/Analytics';
 import { COVID_RESPONSE_SIMULATOR_URL } from 'screens/Tools/Tools';
+import NewHomepage from 'screens/HomePage/New/NewHomepage';
```

Comment:
should we rename `NewHomepage` to just `Homepage`, to eliminate potential confusion? (ie., if this is the new one, what is the other one?)

Reply:
yep, this is just a quick PR to end the a/b experiment and ship the new homepage to all traffic. going to clean up + rename in a separate pr